sdk/storage/azblob/testdata/recordings/Test/TestSASContainerClient.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Entries": [],
```

Comment:
TODO: Why is this empty even though it calls REST API?

Reply:
my guess: sas parameter not being parsed properly somehow